specs/data_structures.md

```diff
@@ -78,37 +86,39 @@ TODO define a transaction format
 | `blockID`    | [BlockID](#blockid)         |             |
 | `signatures` | [CommitSig](#commitsig)`[]` |             |
 
-## Version
-
-| name                | type     | description |
-| ------------------- | -------- | ----------- |
-| `blockchainVersion` | `uint64` |             |
-| `appVersion`        | `uint64` |             |
-
 ## Time
 
-https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/timestamp
+LazyLedger uses the [Google Protobuf Timestamp](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/timestamp) format for timestamps, which represents time as seconds in UTC Epoch Time and nanoseconds.
 
 ## BlockID
 
-| name          | type                            | description |
-| ------------- | ------------------------------- | ----------- |
-| `hash`        | [HashDigest](#hashdigest)       |             |
-| `partsHeader` | [PartSetHeader](#partsetheader) |             |
+The block ID is comprised of two distinct Merkle roots:
+1. The root of the [block header](#header)'s fields, in the order provided in the spec.
+1. The root of the complete [serialized](#serialization) block [split into parts](#todo). This is used at the network layer for securely gossiping parts of blocks.
+
+| name          | type                            | description               |
+| ------------- | ------------------------------- | ------------------------- |
+| `headerRoot`  | [HashDigest](#hashdigest)       | Root of the block header. |
+| `partsHeader` | [PartSetHeader](#partsetheader) | Parts header.             |
```

Comment:
My understanding is that we would definitely remove the `partsHeader`. It is only needed by validators to gossip block data, which validators are not obliged to do (it's optional). And as @musalbas suggested this could be replaced by another gossiping mechanism that uses erasure coded data ([here](https://github.com/LazyLedger/lazyledger-specs/pull/2/files#r376971666) and [here](https://github.com/tendermint/spec/issues/78#issuecomment-584052848)).